examples/qs-only/app.tsx

```diff
@@ -0,0 +1,62 @@
+/** @jsx h */
+import {
+  autocomplete,
+} from '@algolia/autocomplete-js';
+import {
+  createAlgoliaInsightsPlugin,
+} from '@algolia/autocomplete-plugin-algolia-insights';
+import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions';
+import { createLocalStorageRecentSearchesPlugin } from '@algolia/autocomplete-plugin-recent-searches';
+import { Hit } from '@algolia/client-search';
+import algoliasearch from 'algoliasearch';
+import insightsClient from 'search-insights';
+
+import '@algolia/autocomplete-theme-classic';
+
+type Product = {
+  name: string;
+  image: string;
+  description: string;
+  __autocomplete_indexName: string;
+  __autocomplete_queryID: string;
+};
+type ProductHit = Hit<Product>;
+
+const appId = 'latency';
+const apiKey = '6be0576ff61c053d5f9a3225e2a90f76';
+const searchClient = algoliasearch(appId, apiKey);
+insightsClient('init', { appId, apiKey });
+
+const recentSearchesPlugin = createLocalStorageRecentSearchesPlugin({
```

Comment:
We only use `querySuggestionsPlugin` so we can get rid of all the other code.

Reply:
Fixed.